client/src/windows/embeddingMorseSmaleWindow.js

```diff
@@ -89,18 +114,33 @@ class EmbeddingMorseSmaleWindow extends React.Component {
               onDesignSelection={this.props.onDesignSelection}
               activeDesigns={this.props.activeDesigns}
               numberOfWindows={this.props.numberOfWindows}/>
+          </Grid>
+          <Grid item xs={6} className={classes.rightPanel}>
             <MorseSmaleWindow
               dataset={this.props.dataset}
               decomposition={this.props.decomposition}
               numberOfWindows={this.props.numberOfWindows}
               onCrystalSelection={this.props.onCrystalSelection}
               evalShapeoddsModelForCrystal={this.computeNewSamplesUsingShapeoddsModel}/>
-          </ResizablePanels>
+          </Grid>
+        </Grid>
+        <div className={classes.gridlistRoot}>
```

Comment:
something in here might be where the resize issues are originating. Maybe we just need another grid item instead of a div. I'll explore this.

Reply:
I did try another Grid container and things got really ugly... when data was added to the third grid it would expand and eclipse the entire window.

Reply:
It's all working now, including dynamic drawer height based on image size, resizing of browser window, addition of multiple workspace windows, and vertical, wrapped "shape cards".